recipes/libdwarf/all/conanfile.py

```diff
@@ -1,84 +1,103 @@
-from conans import ConanFile, CMake, tools
-import functools
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir, rename
+from conan.tools.build import cross_building
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
 import os
 
-required_conan_version = ">=1.33.0"
-
+required_conan_version = ">=1.53.0"
 
 class LibdwarfConan(ConanFile):
     name = "libdwarf"
     description = "A library and a set of command-line tools for reading and writing DWARF2"
-    topics = ("libdwarf", "dwarf2", "debugging", "dwarf")
+    license = ("LGPL-2.1-only", "GPL-2.0-only", "BSD-2-Clause-Views")
```

Comment:
So long as `dwarfdump isn't part of the package, then you can remove the `GPL-2.0-only` license.
The `dwarfgen` utility is `BSD-2-Clause-Views`, and since we may include that, it should be here.

```suggestion
    license = ("LGPL-2.1-only", "BSD-2-Clause-Views")
```